src/common/selectors/clusterSelectors.ts

```diff
@@ -126,4 +126,7 @@ export const selectSchedulableMasters = (cluster: Cluster): boolean => {
 };
 
 export const isClusterPlatformTypeVM = ({ platform }: Pick<Cluster, 'platform'>) =>
-  !/baremetal|none/.test(platform?.type ?? 'none');
+  SupportedPlatformIntegrations.includes(platform?.type ?? 'none');
+
+// then define the array as...
```

Comment:
- The comment should be removed
- We should unify what is a platform integration, now we have one definition in `useClusterSupportedPlatforms` and another one here. There should be only one.